vector/src/main/java/im/vector/fragments/keybackupsetup/KeybackupSetupStep1Fragment.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.fragments.keybackupsetup
+
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import butterknife.BindView
+import im.vector.R
+import im.vector.fragments.VectorBaseFragment
+
+class KeybackupSetupStep1Fragment : VectorBaseFragment() {
+
+    override fun getLayoutResId() = R.layout.keybackup_setup_fragment
+
+    @BindView(R.id.keybackupsetup_step1_button)
+    lateinit var mButton: Button
+
+    companion object {
+        fun newInstance() = KeybackupSetupStep1Fragment()
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        mButton.setOnClickListener() {
+            val fm = this.activity?.supportFragmentManager
+            fm?.beginTransaction()?.replace(R.id.container, KeybackupSetupStep2Fragment.newInstance())?.addToBackStack(null)?.commit()
```

Comment:
I prefer to see one call per line, for code clarity

Reply:
done